lw-oci-integration/tf_module/README.md

```diff
@@ -0,0 +1,71 @@
+
+## Lacework OCI Integration
+
+The OCI Terraform scripts can be run locally or through an OCI cloud shell. Using the OCI cloud shell is recommended, as it comes with Terraform pre-installed and helps with authentication.
+
+## Prerequisites
+
+1. terraform (recommend >=1.4.6: the version tested)
+
+prerequisites are satisifed by OCI cloud shell.
+
+## Prepare to run Terraform via OCI Cloud Shell
+
+1. Open the OCI cloud shell. For example:
+   1. Log in to the OCI web console.
+   2. Ensure that your home region is selected in the [**Regions**](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/working-with-regions.htm) menu at the top right of the OCI web console. 
+   3. Click the Developer Tools icon next to the **Regions** menu and then [**Cloud Shell**](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cloudshellgettingstarted.htm). The cloud shell should appear with your current region indicated in the prompt at the bottom of the console. 
+2. Create a file main.tf containing:
+    ```
+   module "lacework_oci_cfg_integration" {
+   source = "lacework/config/oci"
+   create = true
+   tenancy_id = var.tenancy_ocid
+   user_email = var.user_email
+   }
+
+   variable "tenancy_ocid" {
+   type = string
+   }
+
+   variable "user_email" {
+   type = string
+   }
+   ```
+3. Create a file terraform.tfvars containing the following, inserting values for tenancy and email:
+    ```
+   tenancy_ocid = ""
+   user_email = ""
```

Comment:
We should probably just get users to set the var default here
```suggestion
   variable "tenancy_ocid" {
   type = string
   default = ""
   }
   variable "user_email" {
   type = string
   default = ""
   }
```

Reply:
Not sure - its not really a "default"?

Reply:
IMO its slightly unusual to declare the vars in the main.tf and then set the value in the tfvars.

We should maybe even just get the variables inline. eg.
```
   module "lacework_oci_cfg_integration" {
   source = "lacework/config/oci"
   create = true
   tenancy_id = "<tenancy_ocid>"
   user_email = "<user_email>"
   }
```